spec/components/geoblacklight/index_map_legend_component_spec.rb

```diff
@@ -8,17 +8,17 @@
   end
 
   it "shows available map text" do
-    expect(rendered).to have_text("Green tile indicates")
+    expect(rendered).to have_text("#7FCDBB tile indicates")
```

Comment:
Can we add a color display_name to the settings? As a user, I don't know what the hex color corresponds to on the page. "Green" is more helpful.

Reply:
I definitely had this thought but I was also wondering if the hex made more sense because this only shows up for screen readers and I would assume anyone using the screen reader might not necessarily be able to match green